css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,42 @@
+* {
+  box-sizing: border-box;
+  font-family: 'DM Sans', sans-serif;
+  font-weight: bold;
+}
+
+.container {
+  padding: 0 10rem;
+}
+
+.row1 {
+  display: flex;
+}
```

Comment:
You can probably just use `text-align` on your heading and remove this class. Avoid `something1`, `something2` in your code if at all possible.